ingestion/src/lib.rs

```diff
@@ -84,11 +91,12 @@ impl FromRef<IngestorState> for Arc<RwLock<(ParamPermitTable, StationPermitTable
 pub struct Datum {
     timeseries_id: i32,
     timestamp: DateTime<Utc>,
-    value: f32,
+    value: ObsType,
 }
 
 pub type Data = Vec<Datum>;
 
+// TODO: benchmark insertion of scalar and non-scalar together vs separately?
```

Comment:
Yes, that would be a good idea. I suspect postgres is smart enough to handle that on its own, but it would be good to know for sure. In any case, it would be easy to put scalar and nonscalar params in different vecs, and it would save us the branching here

Reply:
Also happy to leave this as a todo